src/modules/FtpServer.rb

```diff
@@ -914,6 +915,21 @@ def self.publish_variable(name, type)
     publish function: :OptionsSummary, type: "string ()"
     publish function: :Summary, type: "string ()"
     publish function: :AutoPackages, type: "map ()"
+
+  private
+
+    # Checker for pasive ftp port range
```

Comment:
Typo `passive`, what about # Return whether the min and max passive ftp port range are both valid or not